src/docs/010_manual/60_further_reading.adoc

```diff
@@ -6,102 +6,100 @@ ifndef::imagesdir[:imagesdir: ../images]
 
 include::../_feedback.adoc[]
 
-This chapter lists some additional references to interesting resources.
+Everything you need to know about docToolchain, from the underlying technology to useful resources.
 
 == Underlying Technologies
 
-This project relies on many underlying technologies.
-Here is a lists and some explaining words
+DocToolchain uses many underlying technologies. Explore the list with some resources.
 
 === Markup
 
-* *AsciiDoc* +
-AsciiDoc is our preferred markup language for technical docs
+.AsciiDoc
+[%collapsible]
+==== 
+This is our preferred markup language for technical docs.
+
 ** https://asciidoctor.org/docs/user-manual/[Asciidoctor User-Manual]
 ** https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[AsciiDoc Syntax Quick Reference]
-** Video: https://www.ahus1.de/post/asciidoctor-intro-and-deep-dive["Asciidoctor Deep Dive"] by Alexander Schwartz
-* *Markdown* +
-Since we use jBake as Static-Site-Generator, you can also use Markdown to write your docs.
-** https://github.com/vsch/flexmark-java[flexmark-java] is the Markdown parser and flavour used by jBake
-** if you prefer to use the xref:../015_tasks/03_task_exportMarkdown.adoc[`exportMarkdown`]-Task, then https://github.com/bodiam/markdown-to-asciidoc[markdown-to-asciidoc] library is used. The IntelliJ-Plugin also uses this library.
+** https://www.ahus1.de/post/asciidoctor-intro-and-deep-dive["Asciidoctor Deep Dive Video"] by Alexander Schwartz
+
+====
+
+
+.Markdown
+[%collapsible]
+==== 
+Since we use jBake as Static-Site-Generator, you can write your docs in Markdown too.
```

Comment:
Grammar fix:

Since we use JBake as a static site generator, you can write your docs in Markdown too.